### PR TITLE
MAISTRA-1255: Fix running of builds within the container

### PR DESCRIPTION
**Pull request by knrc**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[X] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by jwendell — CHANGES_REQUESTED** [review]
This `Makefile` is generated from `common-files` repo and should not be manually updated.

My suggestion is to fix this upstream and then we can get the fix on the next micro release.

**Comment by knrc** [comment]
@jwendell We can fix this upstream at a later date, it's needed for us to correctly build this release.

File: Makefile [hunk 1]
```diff
@@ -85,17 +86,17 @@ RUN = $(CONTAINER_CLI) run -t -i --sig-proxy=true -u $(UID):docker --rm \
...
-	-w /work $(IMG)
+	-w $(WORK_DIR) $(IMG)
 else
 $(info Building with your local toolchain.)
 RUN =
 export GOBIN ?= $(GOPATH)/bin
 endif
 
-MAKE = $(RUN) make --no-print-directory -e -f Makefile.core.mk
+MAKE = $(RUN) make $(MFLAGS) --no-print-directory -e -f Makefile.core.mk
```
Comment (jwendell) [inline Makefile:99]: Where does this come from?

**Review by jwendell — COMMENTED** [review]
What problem does this fix?

File: Makefile (hunk 1 above)
Comment (knrc) [reply Makefile:99]: It's a Makefile variable which represents the arguments passed on the command line

**Review by jwendell — APPROVED** [review]

**Comment by knrc** [comment]
@brian-avery Can you please review?

**Comment by knrc** [comment]
Merging manually

**Pull request merged**